README.md

```diff
@@ -1,60 +1,159 @@
 # cloudflare-warp-ingress
 
-Implements an ingress controller using cloudflare-warp tunnel
-to connect a cloudflare-managed url to a kubernetes service.
+Implements a Kubernetes ingress controller using cloudflare-warp tunnel
+to connect a cloudflare-managed URL to a Kubernetes service.
 
-## Deployment
+
+## Getting started
 
 The warp controller will manage ingress tunnels in a single
 namespace of the cluster.  Multiple controllers can exist
 in different namespaces, with different credentials for
 each namespace.
 
-#### Cloudflare certificate
 
-The _cert.pem_ file must be available to the controller as a secret,
-and must be configured in the cluster before the controller can start.
+#### Pre-requirements
+
+To use this, you need:
+- a Cloudflare account (free tier is OK)
+- a zone (domain name) in your Cloudflare account 
+- a Kubernetes cluster
+
+The deployment instructions below use a few YAML files that
+will be used with `kubectl` to create the appropriate resources
+in Kubernetes. You might want to checkout that repository
+to have them handy!
+
+
+#### Get warp credentials
+
+The first step is to obtain the credentials that will
+be used by the controller to authenticate with Cloudflare.
 
+First, install the warp client.
```

Comment:
Same here - s/warp/Warp (proper noun)

Reply:
Good call; I didn't know if you referred to it as "warp" or "Warp" canonically. I've updated it to "Warp" everywhere. 👍🏻